apps/vs-code-designer/src/assets/UnitTestTemplates/TestClassFile

```diff
@@ -75,9 +76,9 @@ namespace <%= LogicAppName %>.Tests
         public async Task <%= WorkflowName %>_<%= UnitTestName %>_ExecuteWorkflow_SUCCESS()
         {
             // PREPARE Mock
-            // var actionMocks = this.actionMocks;
-            // actionMocks["<actionName>"] = new ActionMock(
-            //     "<actionName>",
+            var actionMocks = this.actionMocks;
+            // actionMocks["actionName"] = new ActionMock(
```

Comment:
these i think can remain "<actionName>". Its only in the doc section that angle bracket throws error